code/game/objects/structures/morgue.dm

```diff
@@ -21,7 +21,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
 	density = TRUE
 	anchored = TRUE
 	max_integrity = 400
-
+	pass_flags_self = LETPASSTHROW
```

Comment:
oh right and this needs | PASSSTRUCTURE doesn't it?

Reply:
amended